src/main/java/picard/fingerprint/HaplotypeProbabilitiesFromContaminatorSequence.java

```diff
@@ -142,6 +142,24 @@ public HaplotypeProbabilitiesFromContaminatorSequence merge(final HaplotypeProba
         return this;
     }
 
+    public HaplotypeProbabilitiesFromContaminatorSequence mergeBackground(final HaplotypeProbabilities other) {
+        if (!this.getHaplotype().equals(other.getHaplotype())) {
+            throw new IllegalArgumentException("Mismatched haplotypes in call to HaplotypeProbabilities.merge(): " +
+                    getHaplotype() + ", " + other.getHaplotype());
+        }
+
+        final double[] backgroundLikelihoods = other.getLikelihoods();
```

Comment:
hmmm. can you update the math for this? the LaTeX file is here: `docs/fingerprinting/main.tex`